lib/ObjWriter/.nuget/Microsoft.DotNet.ObjectWriter.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Microsoft.DotNet.ObjectWriter</id>
-    <version>1.0.4-prerelease-00001</version>
+    <version>1.0.6-prerelease-00001</version>
```

Comment:
@kyulee1 Will CoreRT get this updated package if I do that? I need this change in CoreRT as soon as it's merged basically.